### ISSUE TITLE
URL loses trailing slash when using path params

### ISSUE THREAD

**Issue by ThenTech** [usebruno/bruno#2760]
### I have checked the following:

- [X] I use the newest version of bruno.
- [X] I've searched existing issues and found nothing related to my issue.

### Describe the bug

As I encountered in a previous issue of mine (#2528), the interpolation step for the path parameters unintentionally strips the trailing slash.

As seen here (and other instances of this code snippet):
https://github.com/usebruno/bruno/blob/8a9df14e167ff9db082495e32b02bdaf47e79257/packages/bruno-cli/src/runner/interpolate-vars.js#L102-L114

The `pathname` is split on `/`, but then the empty values are filtered out as the URL is rebuilt by prefixing values with `/` instead, causing the trailing slash to be omitted.

A solution could be to add another variable to check for a trailing slash in the original `pathname` and adding it to the final URL:
```javascript
const trailingSlash = url.pathname.endsWith('/') ? '/' : '';
request.url = url.origin + interpolatedUrlPath + trailingSlash + url.search;
```

### .bru file to reproduce the bug
```
meta {
  name: Test
  type: http
  seq: 5
}

get {
  url: {{baseURL}}/some/route/:pk/
  body: none
  auth: inherit
}

params:path {
  pk: test
}

script:pre-request {
  console.log("Pre:", req.url);
}

script:post-response {
  console.log("Post:", req.url);
}
```

### Screenshots/Live demo link

![image](https://github.com/usebruno/bruno/assets/11493646/de0d161c-c09d-4ea6-b2eb-02dce00c9d92)

This is the same image as in #2528, but note the absence of the trailing slash in the URL (in the console and from the error given by Django).

**Comment by sam-briggs-depop** [comment]
Just came here to say I've also seen this issue whilst using Bruno too

**Comment by helloanoop** [comment]
Merged the PR #3065 ! Thanks @sam-briggs-depop 
This will be available in the `v1.32.0` release scheduled to go out next week.
